src/plugin/lib-htmlunit/ivy.xml

```diff
@@ -37,7 +37,8 @@
 
   <dependencies>
     <!-- begin selenium dependencies -->
-    <dependency org="org.seleniumhq.selenium" name="selenium-java" rev="2.44.0" />
```

Comment:
I think I mentioned in the commit comment, lib-htmlunit does not compile in Eclipse without this upgrade. Since I was going to make changes to protocol-htmlunit, I wanted lib-htmlunit to compile.
I will fix plugin.xml, forgot about it.

I will also add a return-null-if-null check to fixHttpHeaders.

Unfortunately, I will only be able to make these two changes on Sunday.

Reply:
Ok, waiting for the update. Thanks!

> lib-htmlunit does not compile in Eclipse

Maybe we could let lib-htmlunit depend on lib-selenium to avoid conflicts and also to save 7-8 MB duplicated jar files in the runtime (or bin package). I'll try whether this works and will open a Jira issue to address this.

Reply:
Updated plugin.xml and fixHttpHeaders as discussed above.